src/methods/tx.rs

```diff
@@ -1,3 +1,29 @@
+//! This module allows you to query the status of a transaction signed by an account_id
```

Comment:
Let's maybe drop the "this module allows" starter.

Example A (this):

![Screenshot_20220309_023327](https://user-images.githubusercontent.com/16881812/157355009-1f3b3951-00e8-49d4-a288-0345fd5ebe1a.png)

Example B (proposed):

![Screenshot_20220309_023502](https://user-images.githubusercontent.com/16881812/157355123-78827f7a-5e42-453a-8576-a5e792699b60.png)

Reply:
Also, the `account_id` could just be an “Account ID”

Reply:
I'm considering dropping the whole “signed by” extension. Feels like an implementation detail.

We could really simplify it to;

```suggestion
//! Queries the status of a transaction.
```